packages/ERTP/src/issuer.js

```diff
@@ -84,7 +93,7 @@ function makeIssuerKit(
   );
 
   const { makeInstance: makePurse } = makeExternalStore('purse', () => {
-    let currentBalance = amountMath.getEmpty();
+    let currentBalance = emptyAmount;
```

Comment:
I see that the use of `makeExternalStore` predates this PR and this PR isn't really changing that. But this is not how I thought `makeExternalStore` works, so I'm having a hard time interpreting this code. Here, we're declaring `currentBalance` as a lexical variable.

Reply:
I don't understand `makeExternalStore` either, but it should go away very soon when we move to the `makeWeakStore` that uses virtual objects.